stock_picking_comment_template/models/stock_picking.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 C2i Change 2 improve - Eduardo Magdalena <emagdalena@c2i.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    """Add text comment"""
+
+    _inherit = "stock.picking"
+
+    comment_template1_id = fields.Many2one('base.comment.template',

```

Comment:
@emagdalenaC2i Should those fields be 'copy=False' ?

Reply:
Yeah, I think they should be preserved on duplication, as you might want to have the same comments. note1/note2 might be arguable to be preserved if they have custom changes though.